core/src/main/java/com/radixdlt/messaging/MessagingModule.java

```diff
@@ -142,7 +142,7 @@ private int maxMessageSize(
     final var maxMessageSizeBaseline =
         Math.max(
             Math.max(
-                REv2TransactionsAndProofReader.MAX_TXN_BYTES_FOR_A_SINGLE_RESPONSE,
+                NodeConstants.MAX_TXN_BYTES_FOR_A_SINGLE_RESPONSE,
```

Comment:
That's true... as a consequence, the `proposalMaxUncommittedTransactionsPayloadSize` is not needed in this method at all.

Lemme add...
```
if (maxUncommittedTransactionsPayloadSize > maxTxnBytesForSingleResponse) {
  log.warn("you want it too high, we gotta use maxTxnBytesForSingleResponse instead");
  return maxTxnBytesForSingleResponse;
}
```
... to the `@ProposalMaxUncommittedTransactionsPayloadSize` provider.
_(since this seems to be our convention)_